tests/package.json

```diff
@@ -13,7 +13,7 @@
     "next": "7.0.2",
     "next-compose-plugins": "^2.1.1",
     "next-manifest": "^1.6.0",
-    "next-offline": "latest",
+    "next-offline": "https://github.com/derekdowling/next-offline#e910c1b",
```

Comment:
We'll want to revert this before merging I guess. This is where a monorepo would be handy 😆

Reply:
yes we will! I'm totally open to a lerna / yarn workspace workflow, know of any similar size repo's that are doing it well?

Reply:
We should move this conversation to an issue though!